Registration Form.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/html">
+  <head>
+    <title>Registration Form</title>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8" />
+    <link
+      href="https://unpkg.com/boxicons@2.0.7/css/boxicons.min.css"
```

Comment:
It is nice that you searched and found an icon library. But Font Awesome should be enough for our task